.github/workflows/ci.yml

```diff
@@ -30,13 +34,12 @@ jobs:
         run: |
           sh ./bootstrap
       - name: Configure
-        run: |
+        run:
           sh ./configure
+            ${{ fromJSON('[ "", "--without-x" ]')[ matrix.without-x ] }}
       - name: Make
-        if: runner.os == 'Linux'
         run: |
           make
-      - name: Make
-        if: runner.os == 'macOS'
+      - name: Show pkg-config files
         run: |
-          make LDFLAGS="$(pkg-config --libs --static libpng xt)"
+          head -n100 *.pc
```

Comment:
Is this `head` just for output? Assuming so since you're applying it to anything with a `.pc` extension.

Reply:
Yes, that's just debugging output since I couldn't find a target to use for `make test`. Should also help make sure that disabling X11 means the `pkg-config` does not have a `Requires:` with `x11` in the list.